packages/flame/lib/src/components/position_component.dart

```diff
@@ -98,6 +101,19 @@ class PositionComponent extends Component
   final NotifyingVector2 _size;
   Anchor _anchor;
 
+  /// The angle where this component is looking at in its
+  /// default state, i.e. when [angle] is equal to zero.
+  double _nativeAngle;
+  double get nativeAngle => _nativeAngle;
+  set nativeAngle(double value) {
+    _nativeAngle = value;
+    _nativeDirection.setFrom(Vector2(0, -1)..rotate(_nativeAngle));
+  }
+
+  /// The forward direction for this component when it is in its
+  /// default state, i.e. when [angle] is equal to zero.
+  final Vector2 _nativeDirection;
```

Comment:
I don't think we want another `Vector2` field for this since it makes the class heavier, I also think there is a rotate method on `Vector2` so one could just do `Vector2(0, -1)..rotate(nativeAngle)` to get this.

Reply:
Wouldn't doing this in `lookAt` result in creation of a new Vector2 object? Chances of people calling `lookAt` in update are right.

Reply:
Removed _nativeDirection.